src/web/components/SubNav/SubNav.tsx

```diff
@@ -1,87 +1,207 @@
-import React, { Component, createRef } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { css, cx } from 'emotion';
 
 import { GuardianLines } from '@frontend/web/components/GuardianLines';
+import { text } from '@guardian/src-foundations/palette';
+import { palette } from '@guardian/src-foundations';
```

Comment:
I know this is just copied code but it'd be great to fix these direct uses of `palette` as we see them 🙏